devtools/packaging/scripts/osx/build.sh

```diff
@@ -37,7 +38,7 @@ fi
 mkdir build
 cd build
 cmake ../openmm $CMAKE_FLAGS
-make -j4 all DoxygenApiDocs sphinxpdf
+make -j4 all PythonInstall C++ApiDocs PythonApiDocs sphinxpdf
```

Comment:
> you moved make PythonInstall before make install in peastman@46fb5c5, which is wrong. 

That was just a little while ago, and was an attempt to fix something that was already broken.  How did it get broken in the first place.

> And in <7.0, make PythonApiDocs didn't require make PythonInstall.

Yes it did.  It might not have produced the same error, but it definitely didn't produce correct documentation.  It involved running Doxygen on documentation that was generated by `make PythonInstall`.